tests/helpers.py

```diff
@@ -1,5 +1,6 @@
 import pickle
```

Comment:
tests/wrappers.py?

there will probably be more of these.

Reply:
Good idea. I'll move it.